### PR TITLE
Assignment 2 Task

### PR DESCRIPTION
**Pull request by suyash929**
@sriram15690 , please review assignment of JavaScript

### CODE REVIEW

File: Assignment2/basicOp.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function basicOp(operation, value1, value2)
+{
+  // Code
+  return operation === '+' ? value1 + value2 : operation === '-' ? value1-value2 : operation === '*' ? value1*value2 : operation === '/' ? value1/value2 : 0
```
Comment (sriram15690) [inline Assignment2/basicOp.js]: when there are 4 cases, you can use "switch" here

File: Assignment2/printArray.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function printArray(array){
+  //show me the code!
```
Comment (sriram15690) [inline Assignment2/printArray.js]: can be a simple arrow function: 

const printArray = (data) => (data.join(""));

File: Assignment2/oppositeNumber.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+function opposite(number) {
+  //your code here
```
Comment (sriram15690) [inline Assignment2/oppositeNumber.js]: remove // your code here comment

File: Assignment2/printArray.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+function printArray(array){
```
Comment (sriram15690) [inline Assignment2/printArray.js]: argument name shouldn;t be "array". It should be adjective like "userList" or. "productList" or "userArray"

File: Assignment2/rentalCarCost.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+function rentalCarCost(d) {
+  // Your solution here
+  return d >= 7? 40*d - 50 : d >=3 ? 40*d - 20 : 40*d
```
Comment (sriram15690) [inline Assignment2/rentalCarCost.js]: this is okay, but if, else if, else is more readable. but its fine for now.

**Review by sriram15690 — CHANGES_REQUESTED** [review]

File: Assignment2/printArray.js (hunk 4 above)
Comment (suyash929) [reply Assignment2/printArray.js]: Actually we were given this exercise on codewars platform there they mentioned it as array and they asked to code below.
I will change this.

File: Assignment2/basicOp.js (hunk 1 above)
Comment (suyash929) [reply Assignment2/basicOp.js]: ok I will make this change

File: Assignment2/printArray.js (hunk 2 above)
Comment (suyash929) [reply Assignment2/printArray.js]: Actually we had to do task on the basis what we learnt through session. In session trainer didn't mentioned about arrow function so , that's why I haven't used it

File: Assignment2/oppositeNumber.js (hunk 3 above)
Comment (suyash929) [reply Assignment2/oppositeNumber.js]: I will remove this coment

File: Assignment2/rentalCarCost.js (hunk 5 above)
Comment (suyash929) [reply Assignment2/rentalCarCost.js]: Ok I will do changes.

File: Assignment2/printArray.js (hunk 4 above)
Comment (sriram15690) [reply Assignment2/printArray.js]: ok

**Review by sriram15690 — APPROVED** [review]

**Pull request merged**